libs/design/breadcrumb/src/breadcrumb-item/breadcrumb-item.directive.ts

```diff
@@ -1,21 +1,34 @@
 import {
   Directive,
+  ElementRef,
   HostBinding,
-  Input,
+  Renderer2,
 } from '@angular/core';
 
 @Directive({
   selector: 'li[daffBreadcrumbItem]',
-  standalone: true,
 })
 export class DaffBreadcrumbItemDirective {
+  /**
+   * @docs-private
+   */
   @HostBinding('class.daff-breadcrumb__item') class = true;
 
-  @HostBinding('attr.aria-current') get ariaCurrent() {
-    return this.active ? 'page' : 'false';
-  }
+  private _active = false;
 
-  /** Whether or not the breadcrumb item is active */
-  @Input() @HostBinding('class.active') active = false;
+  constructor(
+    private elementRef: ElementRef,
+    private renderer: Renderer2,
+  ) {}
 
+  /** Called by the DaffBreadcrumbComponent to set the active state */
+  setActive(value: boolean) {
+    this._active = value;
+    if (value) {
+      this.renderer.addClass(this.elementRef.nativeElement, 'active');
```

Comment:
I think it makes sense to keep the original implementation and just remove the `input`.

Reply:
updated